core/toolbox/toolbox.js

```diff
@@ -558,13 +558,80 @@ Blockly.Toolbox.prototype.wouldDelete = function(element, _couldConnect) {
   if (element instanceof Blockly.BlockSvg) {
     var block = /** @type {Blockly.BlockSvg} */ (element);
     // Prefer dragging to the toolbox over connecting to other blocks.
-    this.wouldDelete_ = !block.getParent() && block.isDeletable();
+    this.updateWouldDelete_(!block.getParent() && block.isDeletable());
   } else {
-    this.wouldDelete_ = element.isDeletable();
+    this.updateWouldDelete_(element.isDeletable());
   }
   return this.wouldDelete_;
 };
 
+/**
+ * Handles when a cursor with a block or bubble enters this drag target.
+ * @param {!Blockly.IDraggable} _dragElement The block or bubble currently being
+ *   dragged.
+ */
+Blockly.Toolbox.prototype.onDragEnter = function(_dragElement) {
+  this.updateCursorDeleteStyle(true);
+};
+
+/**
+ * Handles when a cursor with a block or bubble exits this drag target.
+ * @param {!Blockly.IDraggable} _dragElement The block or bubble currently being
+ *   dragged.
+ * @override
```

Comment:
If you specify `@override` you don't have to repeat the jsdoc unless you want to. It's not really needed here since you're not using the params

Should the other methods also be marked `@override`?

Reply:
Yeah, the other ones should be `override`.